### PR TITLE
python3Packages.afdko: 3.9.0 -> 3.9.2

### PR DESCRIPTION
**Pull request by onny**
###### Description of changes

Update python packages

- afdko 3.9.0 -> 3.9.2 https://github.com/adobe-type-tools/afdko/blob/develop/NEWS.md#392-released-2023-01-10
- lxml 4.9.1 -> 4.9.2 https://github.com/lxml/lxml/compare/lxml-4.9.1...lxml-4.9.2
- fonttools 4.34.4 -> 4.38.0 https://github.com/fonttools/fonttools/releases
 
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/afdko/default.nix [hunk 1]
```diff
@@ -68,14 +85,17 @@ buildPythonPackage rec {
     tqdm
   ];
 
+  preBuild = ''
+    # Use system libxml2
+    export FORCE_SYSTEM_LIBXML2=ON
+  '';
```
Comment (sternenseemann) [inline pkgs/development/python-modules/afdko/default.nix:91]:
```suggestion
  FORCE_SYSTEM_LIBXML2 = true;
```

**Review by sternenseemann — COMMENTED** [review]
I expect this needs to go to staging.

**Comment by sternenseemann** [comment]
@ofborg build noto-fonts-emoji

**Comment by onny** [comment]
@sternenseemann Thank you, done :)

**Comment by sternenseemann** [comment]
@ofborg build python3Packages.afdko.tests noto-fonts-emoji python3Packages.pygments

File: pkgs/development/python-modules/afdko/default.nix [hunk 2]
```diff
@@ -14,13 +33,13 @@
 
 buildPythonPackage rec {
   pname = "afdko";
-  version = "3.9.0";
+  version = "3.9.2";
```
Comment (mweinelt) [inline pkgs/development/python-modules/afdko/default.nix:36]:
```suggestion
  version = "3.9.2";
  format = "pyproject";
```

Move the format up here.

File: pkgs/development/python-modules/afdko/default.nix [hunk 3]
```diff
@@ -14,13 +33,13 @@
...
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "1fjsaz6bp028fbmry6fzfcih78mdzycqmky1wsz5y0bg4kfk4shh";
+    sha256 = "sha256-3JEnQbS4CtZEqAb+/ngqkO+nv9SZ0Zi8sPJLyW+tQ9w=";
```
Comment (mweinelt) [inline pkgs/development/python-modules/afdko/default.nix:42]:
```suggestion
    hash = "sha256-3JEnQbS4CtZEqAb+/ngqkO+nv9SZ0Zi8sPJLyW+tQ9w=";
```

File: pkgs/development/python-modules/afdko/default.nix [hunk 4]
```diff
@@ -103,7 +117,7 @@ buildPythonPackage rec {
 
   meta = with lib; {
     description = "Adobe Font Development Kit for OpenType";
```
Comment (mweinelt) [inline pkgs/development/python-modules/afdko/default.nix:119]:
```suggestion
    changelog = "https://github.com/adobe-type-tools/afdko/blob/${version}/NEWS.md";
    description = "Adobe Font Development Kit for OpenType";
```

File: pkgs/development/python-modules/fonttools/default.nix [hunk 5]
```diff
@@ -24,15 +24,15 @@
 
 buildPythonPackage rec {
   pname = "fonttools";
-  version = "4.34.4";
+  version = "4.38.0";
```
Comment (mweinelt) [inline pkgs/development/python-modules/fonttools/default.nix:27]:
```suggestion
  version = "4.38.0";
  format = "setuptools";
```

Add explicit format.

File: pkgs/development/python-modules/fonttools/default.nix [hunk 6]
```diff
@@ -24,15 +24,15 @@
...
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner  = pname;
     repo   = pname;
     rev = "refs/tags/${version}";
-    sha256 = "sha256-GwbcrDsfxs5qRQJozhK/+n3W3NlO39g7pzxL9iIiDfU=";
+    sha256 = "sha256-cdZI2kwR3zzS6eiiXGpeHIp+kgPCPEsTOSTV60pODTM=";
```
Comment (mweinelt) [inline pkgs/development/python-modules/fonttools/default.nix:35]:
```suggestion
    hash = "sha256-cdZI2kwR3zzS6eiiXGpeHIp+kgPCPEsTOSTV60pODTM=";
```

File: pkgs/development/python-modules/lxml/default.nix [hunk 7]
```diff
@@ -8,13 +8,13 @@
 
 buildPythonPackage rec {
   pname = "lxml";
-  version = "4.9.1";
+  version = "4.9.2";
```
Comment (mweinelt) [inline pkgs/development/python-modules/lxml/default.nix:11]:
```suggestion
  version = "4.9.2";
  format = "setuptools";
```

File: pkgs/development/python-modules/lxml/default.nix [hunk 8]
```diff
@@ -8,13 +8,13 @@
...
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "refs/tags/lxml-${version}";
-    sha256 = "sha256-5MJw3ciXYnfctSNcemJ/QJGKAaYpadvdbFhkc8+pmPM=";
+    sha256 = "sha256-IHuTlcDbrZHvS6Gtx48IkznVU+9WxZT9XHUZf8M1WOE=";
```
Comment (mweinelt) [inline pkgs/development/python-modules/lxml/default.nix:17]:
```suggestion
    hash = "sha256-IHuTlcDbrZHvS6Gtx48IkznVU+9WxZT9XHUZf8M1WOE=";
```

**Review by mweinelt — CHANGES_REQUESTED** [review]
Some nits.

**Review by sternenseemann — APPROVED** [review]

**Pull request merged**